File: snow_gushi/question3/Cryptography.py
class Cryptography():
    def encrypt(self, numbers):
        numbers = list(numbers)
        min_number = min(numbers)
        multiply_number = 1 
        once_plus = False
        for number in numbers:
            if number == min_number and once_plus is False:
               number += 1
               once_plus = True
            multiply_number *= number
        return multiply_number 

File: chooblarin/question03/Cryptography.java
public class Cryptography {
    public long encrypt(int[] numbers) {
        long m = 0;
        for (int i = 0; i < numbers.length; i++) {
            long p = 1;
            for (int j = 0; j < numbers.length; j++) {
                if (i == j) {
                    p *= numbers[j] + 1;
                } else {
                    p *= numbers[j];
                }
            }
            m = Math.max(m, p);
        }
        return m;
    }
}
